app/models/review_app.rb

```diff
@@ -68,4 +68,9 @@ def computed_environment
       {name: "BARCELONA_REVIEWAPP_DOMAIN", value: domain}
     ]
   end
+
+  # replace underscores if present - not valid as subdomain
+  def sanitized_subject
+    subject.gsub('_', '-')
```

Comment:
If we are going to do this lets just remove the validation and go ham `gsub(/[^a-z0-9]+/, '-')`

What do you think?

Reply:
this would let nonsense through to be interpreted as hyphens (eg. #!$%). If that's acceptable I think its okay